website/docusaurus.config.js

```diff
@@ -3,18 +3,18 @@ module.exports = {
   title: 'Next.js API Decorators',
   tagline:
     'Collection of decorators to create typed Next.js API routes, with easy request validation and transformation.',
-  url: 'https://storyblok-toolkit.vercel.com',
+  url: 'https://next-api-decorators.vercel.com',
   baseUrl: '/',
   onBrokenLinks: 'throw',
   onBrokenMarkdownLinks: 'warn',
   favicon: 'img/favicon.ico',
-  organizationName: 'storyofams',
+  organizationName: 'instantcommerce',
   projectName: 'next-api-decorators',
   themeConfig: {
```

Comment:
Maybe you can [prefer-color-scheme](https://docusaurus.io/docs/api/themes/configuration#respectPrefersColorScheme) and default to `black` ?

Think it would be nice to have the dark mode as default as it does have the branding and looks better IMO than the grayscales (also not sure if we should put more time/effort into updating the color scheme).

If we do choose to spend more time on it, I'd say we just get a vote on what theme we like and roll with that for now?

Reply:
it'd be great if it could detect the system theme, but don't know if there is an option like that

Reply:
@ggurkal is it easy to default to dark mode? In that case I would suggest we do that.

Reply:
yes. it is quite easy. so do we go for dark mode rather than detecting the system theme?

Reply:
Yeah going for dark sounds good to me

Reply:
@ggurkal my link shows how to add the media query preference to fall back to the theme